.github/workflows/version.yml

```diff
@@ -0,0 +1,35 @@
+---
+name: Version
+
+run-name: Cut ${{ github.event.inputs.version }}
+
+on:
+  workflow_dispatch:
+    inputs:
+      version:
+        description: Version to cut
+        required: true
+
+jobs:
+  tag:
+    name: Tag
+    runs-on: ubuntu-latest
+    timeout-minutes: 30
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v4
+        with:
+          token: ${{ secrets.GH_TOKEN }}
+      - name: Import GPG key
+        uses: crazy-max/ghaction-import-gpg@v6
+        with:
+          git_user_signingkey: true
+          git_commit_gpgsign: true
+          git_committer_name: ${{ secrets.GIT_USER_NAME }}
+          git_committer_email: ${{ secrets.GIT_USER_EMAIL }}
+          gpg_private_key: ${{ secrets.GPG_PRIVATE_KEY }}
+          passphrase: ${{ secrets.GPG_PASSPHRASE }}
+      - name: Setup
+        uses: ./.github/actions/setup
+      - name: Cut ${{ github.event.inputs.version }} version
+        run: npm version --sign-git-tag=true ${{ github.event.inputs.version }}
```

Comment:
Ideally we could use some php tool here like composer and not rely on npm, but this is a good step in that direction. We can do this for now and replace it later. See for example how bundler and poetry handle this in our ruby and python repos